src/main/java/seedu/address/commons/core/LogsCenter.java

```diff
@@ -60,53 +59,43 @@ public static <T> Logger getLogger(Class<T> clazz) {
     }
 
     /**
-     * Adds the {@code consoleHandler} to the {@code logger}. <br>
-     * Creates the {@code consoleHandler} if it is null.
+     * Creates and returns a base logger with the default configuration and containing a {@code ConsoleHandler}
+     * and a {@code FileHandler}.
+     * The base logger is the parent of all other loggers. Changing the level of the base logger will change
+     * the level of all other loggers if they are untweaked. Similarly, all loggers created from this class will
+     * inherit the handlers of the base logger.
      */
-    private static void addConsoleHandler(Logger logger) {
-        if (consoleHandler == null) {
-            consoleHandler = createConsoleHandler();
-        }
-        logger.addHandler(consoleHandler);
-    }
+    private static Logger getBaseLogger() {
+        Logger baseLogger = Logger.getLogger("ab3");
```

Comment:
The current role of this method doesn't feel right. It returns something that is meant to be used as the baseLogger but it has no control over how it will be used.
The following feels like a better fit for what the method does.
```suggestion
    private void getBaseLogger() {
        baseLogger = Logger.getLogger("ab3");
```
It can be called inside the init() method. What do you think?

Reply:
I would still prefer the original method, as `init()` is only called after the config file is being read. 
This will cause the `baseLogger` to be uninitialized before then. 

To prevent this, `LogsCenter` might need to switch to using a getter method to retrieve the `baseLogger` (initialize it if it is not defined) which is not really ideal since `baseLogger` itself is a private attribute of `LogsCenter`

Reply:
I don't think that is a major problem, compared to an abstraction that doesn't sound right. But if you want something to execute earlier, you can use a `static` code block. See `CommandTestUtil.java` for an example. Would that work? e.g.,
```
static {
    setBaseLogger();
}
```